src/styles/utils/variables.scss

```diff
@@ -0,0 +1,10 @@
+$border-radius-size: 5px;
+$title_color:#060b35;
+$button-color: #00acdc;
+$card-background-color:#fff;
+$btn-background: var(--blue-accent, $button-color);
+$btn-border: 1px solid var(--blue-accent, $button-color);
+$title-color-value: var(--main-accent,$title_color);
+$desc-color: var(--secondary, #616070);
+$star-number: 1,2,3,4;
+$header-color: #00acdc;
```

Comment:
you don't use most of them more than one time, therefore you should delete them, 11'th point in checklist